frontend/viewer/src/lib/components/field-editors/select.svelte

```diff
@@ -129,6 +135,11 @@
             {label}
           </CommandItem>
         {/each}
+        {#if renderedOptions.length < filteredOptions.length}
+          <div class="text-muted-foreground text-sm px-2 py-1">
+            {$t`Refine your filter to see more...`}
+          </div>
+        {/if}
```

Comment:
Semantic domains are a UI challenge. I feel like we might need a dedicated component for them at some point. But this is probably good enough for now, as long as the search is sensible so users can easily find the semantic domain they're looking for. (Probably want to search in semdom questions and descriptions, not just in semdom title... but I'm not sure about that. Should look at what FLEx does.)

Reply:
Yeah, there's definitely room for improvement. For now we're mostly just trying to have at least the same feature-set as the only UI.

Reply:
FLEx actually has 2 alternative ways for searching:
1) Inline does what we do, I think
2) In a dialog, they search other (somewhat hidden) texts.